### PR TITLE
Task 3

### PR DESCRIPTION
**Pull request by larsmr**
Task 3: 
Containing simplified methods for calculating added mass, and undamped and uncoupled natural period in heave for a barge and a cylinder

### CODE REVIEW

**Comment by EBGlom** [comment]
I suggest to rather make a `FloatingCylinder` and `FloatingBarge` class with the same methods. Rename `added_mass `to `estimate_added_mass` and let this method store the calculated added mass  `self.added_mass` . Finally, let `natural_period_heave ` call `estimate_added_mass` if the added mass is not given as input.

Perhaps the second question should be related to a barge, as you have added that formula?

File: README.md [hunk 1]
```diff
@@ -47,8 +47,15 @@ This class can be used to answer he following questions:
...
 
+The task 3 Python file contains a simplified ```NaturalPeriod``` class that can return added mass and natural period for a cylinder and a barge in accordance with approximated methods
+This class can be used to answer the following questions:
+ - a) What is the added mass of a cylinder with diameter = 100m
+ - b) What is the natural period of the same cylinder with mass = 200,000t. Use the added mass determined in a) (if you didn't solve a), use a33=300,000t)
+
+Be aware that the ```NaturalPeriod``` class and its methods expect and return SI units.
 ### 4)
 
 ### 5)
```
Comment (EBGlom) [inline README.md]: I move task 1) to task5) in the master branch. Readme.md must be merged before the pull request can be approved

File: tasks/task3.py [hunk 2]
```diff
@@ -0,0 +1,59 @@
...
+        if self.geometry in 'cylinder' and diameter is not None:
+            return round(1/3 * self.rho * diameter**3, 1)
+        elif self.geometry in 'barge' and (width and draft and length) is not None:
+            return round(0.8 * self.rho * width * draft * length, 1)
+        else:
+            raise ValueError(f"Invalid combination of geometry: {self.geometry} and characteristic lengths. "
+                             f"If geometry is barge give: width, draft and length. "
+                             f"If geometry is cylinder: give diameter")
+
+    def natural_period_heave(self, mass, waterplane_area, added_mass):
```
Comment (EBGlom) [inline tasks/task3.py]: I suggest to make this part of a `SimplifiedFloater` class which is inherited to `SimplifiedCylinder` and `SimplifiedBarge` classes with their own `estimate_added_mass` metohds

**Comment by larsmr** [comment]
Implemented suggested changes for task 3

**Pull request merged**